Sources/PluginLibrary/swift-options.pb.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 import SwiftProtobuf
 
 
-let Google_Protobuf_FileOptions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
+let Extensions_appleSwiftPrefix = ProtobufGenericMessageExtension<ProtobufOptionalField<ProtobufString>, Google_Protobuf_FileOptions>(protoFieldNumber: 50138, protoFieldName: "apple_swift_prefix", jsonFieldName: "appleSwiftPrefix", swiftFieldName: "appleSwiftPrefix", defaultValue: "")
```

Comment:
Ha, I picked the wrong thing to look at when thinking about this initially.  `swift-options.proto` has no proto package and no Swift prefix, so that is why this remained _simple_ name.  Just checked all generated code for extensions in the `Reference` directory, they names are fully scoped.  There is one other test file with no proto package or prefix, so the names didn't change there either (as expected).  So I think we're good here.